### PR TITLE
facehuggers can now infect IPCs

### PR DESCRIPTION
**Pull request by GDNgit**
## What Does This PR Do
Facehuggers can now infect IPCS, has a different stage progression. Instead of dealing toxin damage it deals some brute and burn, text and emotes also changed.

## Why It's Good For The Game
IPCs not being able to be infected by facehuggers leads to a couple of odd things:
1. It's not really good for the IPC because they are pretty much dead anyways if they get facehugged, so the bonus is minimal at best. 
2. It's not really good for a xeno to get an IPC target, so xenos often just won't bother which leads to either A. Permakill the IPC, or B. Ignore the IPC
3. It encourages a lot of IPC permakilling, get killed as an IPC on a xeno round and you're gonna end up dead for the rest of it, and you lose out on a xeno roll yourself by not being infected. 

It really does just lead to a bunch of super "meh" moments where an antag just ignores a group of people, as a xeno it sucks and if you're an IPC it's still pretty meh? 

_lore_ reason for why this can happen is that the larva effectively rips apart the IPCs internals to cause internal heating damage, serving as a makeshift incubation chamber for the larva

Admins can also assign any organ to larva now which will be used in a funny way I bet

## Testing
Compiled, ran, infected and removed larva

## Changelog
:cl:
tweak: IPC can now be infected with alien larva
/:cl:

### CODE REVIEW

**Comment by Bkulli-VI** [comment]
As an IPC player, this is kind of a lose/lose kind of change.

Mechnics wise this is adding another logically inconsistent thing to the IPC package as a whole.  IPCs are already poorly suited for combat in general much less to xenomorphs.   While I can understand making them a reasonable target for an antag a respectable goal.  there is more to think about to have it be even close to internally consistent logic.

In other media, Xenomorphs just ignore robots and machines unless they present themselves to a threat to the hive.  Likely a weird thing to add to the game in some way, and I assume an undesired outcome.  But unless our lore is different from other Xenomorphs it doesnt really make sense why why would need to bother.   and forgive me if I assume the two lore are mostly the same, as any key differences are not readily noted.

Facehuggers stunning someone is described as a chemical method, which IPCs are largely immune to (expect those ones that dont). Which is used to largely secure the implantation process.   Another step in this process is the proboscis, which in organics they inject into the mouth.  Which IPCs also lack, which they would be notified if they say try to ingest food or pills. So another strike against the logic of it.

The next step gets a bit grey in turns of what you listen to and what the goal and function of a facehugger.  the example idea in the PR implies they take in heat to gestate.  Other media Xenomorphs imply this is a largely biological and parasitical relationship to put it bluntly.  Some point to using biochemicals from a stomach to feed and grow the larva, other implying it takes in dna of the host to adapt the xenomorph to the surrounding environment.  Once more IPC lack chemical reactions of most shorts, and they also lack dna (unless its a feature that can be used against them, such as dna scanners then it exist)

Thirdly Larvae crew and eat there way out leading to the famous chest bursting.  On organics the heart is destroyed in this part of the process.  I assume story wise it is eaten, though game mechanics I can only guess as to why.  IPC functional heart is a battery, a question of how or why it would eat one would be anyone guess at that point.  IM sure it makes sense to someone.


Those are rather important story reasons. give me a night or two and I can probably think of more.  unless we magic wand a reason or one has already been properly ensorcelled into existence of our xenos just work different.  

From a gameplay perspective, all this really does is streamline some stuff for Xenos making it easier and simpler.  IPCs even without infection melt in any kind of prolonged exposure to direct combat, even if they somehow got a barring and fought back reasonably well, they are likely slow, about to lose a limb, and if infectible just an easy pop tick for a xeno.   I could vaguely understand some kind of balance argument why they should be infectible.  But it would be another weird thing that just works when logically and settings wise it shouldn't.  IPCs already have loads of exceptions to things they are immune to or parts they don't have, that exist only when it would be a disadvantage to them, and this would join in on that list.

Personally I would change xenos so it works like a medhud vs Diag hud, they dont see machines as easily thus to simulate or imply they are not immediate targets.  and if they want to attack them, odds are they wont stand by while friends are being attacks and will step in, getting in the way of the hive at that point and become as they say valid.   Though I assume there is some kind of design headache with that idea.

TLDR, dissagree with this PR

**Comment by Gatchapod** [comment]
Yesterday I have seen IPCs suicide-charging xenomorphs, likely because they know losing costs crew basically nothing.
Killing an armed IPC is obnoxious, especially if they carry a big weapon like an immolator. Killing them takes a long while, too. Xenomorphs do not actually deal high damage, so you're just standing there, slashing a Lego man.

Actual Alien lore matters little. Xenomorphs here can't afford to ignore non-organics. It's a cool idea, but not realistically achievable, even due to a simple fact that xenomorph players can't immediately recognize non-organics like the Alien ones do, so they might attack IPCs by pure accident.

Overall, this PR seems like a reactionary one, since yesterday I complained how my entire hive was wiped and I had to rebuild it as a solo queen. I ran into IPCs twice before I could find an eligible host. However, that was a well-known case of shit luck, not actual balance issue.
While it's likely true that capturing IPCs should have _some_ benefit for xenomorphs, I'm not sure if shoehorning them into being valid hosts is a good solution.

**Comment by Carthusia** [comment]
Overall, not a fan of this at all.

They arnt organic, theres literaly nothing to allow it to grow inside it, theres hardly even a mouth on them, if at all.

If IPCs were made viable hosts, then youd have to also add borgs, and other inorganic things controlled by players.

The lore may not matter hugely, but this is one thing that is universal. 

Live organic hosts are needed

**Comment by 1080pCat** [comment]
> If IPCs were made viable hosts, then youd have to also add borgs, and other inorganic things controlled by players.

Me and my fellow beno hunters kidnapping the ai for our face huggers.

And this PR is.. hmm, something i both agree and disagree with, something would have to be done about ipcs just being borgs with guns towards xenos, but this really isn't the way imo, maybe give xenos a damage multiplier to ipcs? as ipcs can just run in with no fear as if they die, they die, doesn't add more xenos for crew to fight, and they can be revived rather easily and cheaply..

**Comment by OctusGit** [comment]
Yeah this PR ain't it chief. It just seems silly to make IPCs a non organic host compatible with xeno larva. It doesn't make any sense and it takes away from the differences of playing a human vs an IPC. 

Now we just made them more human which hurts the purpose of playing a different species. It generalizes them and this is in the direction im strongly against.

You play a species with different personal challenges, this one shouldn't be exempted.

**Comment by zzzmike** [comment]
Echoing what others have said, but it just doesn't make sense for a machine to be able to be xeno infected.

**Comment by hal9000PR** [comment]
3 balance team objections for the most of the reasons listed by commenters.
Zero community or balance team in support, filing as closed.
